pymtl3/passes/rtlir/rtype/RTLIRType.py

```diff
@@ -257,10 +257,12 @@ def _gen_parameters( s, obj ):
     arg_names = s.argspec.args[1:]
     assert s.argspec.varargs is None, "varargs are not allowed for construct!"
     assert s.argspec.keywords is None, "keyword args are not allowed for construct!"
-    kwargs = ()
+    # Yanghui : I commented out param_dict related stuff, since we are not
+    # adding extra key word arguments with set_param.
+    # kwargs = ()
 
```

Comment:
@ptpan I don't think this part is necessary since `set_param` can only set the value for existing keyword arguments. It won't add extra constructor arguments. Can you take a look at this?

Reply:
Right these are not needed since we can rely on `_dsl.kwargs` to get all keyword argument values.